metadata-ingestion/src/datahub/ingestion/extractor/schema_util.py

```diff
@@ -251,12 +263,18 @@ def __init__(
             converter: "AvroToMceSchemaConverter",
             description: Optional[str] = None,
             default_value: Optional[str] = None,
+            meta_mapping_processor: Optional[OperationProcessor] = None,
+            schema_tags_field: Optional[str] = None,
+            tag_prefix: Optional[str] = None,
         ):
             self._schema = schema
             self._actual_schema = actual_schema
             self._converter = converter
             self._description = description
             self._default_value = default_value
+            self._meta_mapping_processor = meta_mapping_processor
+            self._schema_tags_field = schema_tags_field
+            self._tag_prefix = tag_prefix
```

Comment:
we can get these using `converter._something` right? is there a reason to pass them explicitly?

Reply:
that makes sense. there is no reason to pass them explicitly.